app/cdap/components/AddTransformation/index.tsx

```diff
@@ -28,28 +28,49 @@ import {
 } from 'components/AddTransformation/types';
 import { getDataQuality } from 'components/AddTransformation/CircularProgressBar/utils';
 import { multipleColumnSelected, ADD_TRANSFORMATION_PREFIX } from 'components/AddTransformation/constants';
+import FunctionNameWidget from 'components/AddTransformation/FunctionNameWidget';
+import SelectColumnsWidget from 'components/AddTransformation/SelectColumnsWidget';
+import SelectedColumnCountWidget from 'components/AddTransformation/SelectedColumnCountWidget';
+import ButtonWidget from 'components/AddTransformation/ButtonWidget';
+import { getDirective } from 'components/AddTransformation/utils';
 
 export default function({
-  directiveFunctionSupportedDataType,
+  transformationFunctionSupportedDataType,
```

Comment:
rename this to something better
transformationFunctionSupportedDataType

Reply:
whya re we using transformationFunction in the variable name, is it so generalised
is it not specific to som transformation

if it is general then remove the keyword function from variable name
and make it more proper, it can be transformationDataTypes

Reply:
Done